plenum/test/util/test_common_util.py

```diff
@@ -24,15 +24,15 @@ def test_friendly_exception_formatting_exception_without_reason_field():
     assert formatted_exception == '{}'.format(ex)
 
 
-def test_friendly_exception_formatting_exception_with_reason_field():
+def test_friendly_exception_formatting_exc_with_str_overload():
     """
         Check if the `friendyEx` is formatting exceptions with a reason field correctly
     """
     ex = InsufficientSignatures(1, 3)
 
     formatted_exception = friendlyEx(ex)
 
-    assert formatted_exception == ex.reason
+    assert formatted_exception == '{}'.format(ex)
```

Comment:
I think `assert formatted_exception == ex.reason` is more clear for checking that we expect `reason`. `'{}'.format(ex)` doesn't show us a difference in str overload.

Reply:
Agree, I made the needed changes.